core/src/main/java/io/ddf/analytics/AStatisticsSupporter.java

```diff
@@ -329,7 +332,7 @@ public void setThirdQuantile(double mThird_quantile) {
 
     mLog.info("Column type: " + colType);
     List<Double> pValues = Arrays.asList(percentiles);
-    Pattern p1 = Pattern.compile("(big|small|tiny|int)");
+    Pattern p1 = Pattern.compile("(big|small|tiny|int|long)");
```

Comment:
I've made a test using the this code
'''
Pattern p1 = Pattern.compile("(big|small|tiny|int|long)");
System.out.println(p1.matcher("small").matches());
System.out.println(p1.matcher("smallint").matches());
System.out.println(p1.matcher("int").matches());
'''
The result is true, false, true, respectively. Thus, that line of code (Line 335) might not work correctly.